### DISCUSSION TITLE
Where am I running

### DISCUSSION THREAD

**Discussion by jorigel** [ltsp/ltsp#904] [Q&A] [answered]
Hi Alkis,
is there a simple way to decide in a terminal if I'm running in the configuring virtualbox or the running ltsp?  _ltsp info_ doesn't show a difference if called from both hemispheres.

thank You for giving support like always :)

**Comment by alkisg** [comment]
blkid or lsblk, and check if /dev/loop0 is squashfs or MBR/GPT.

**Comment by jorigel** [comment]
Hi,
this doesn't work for me, because it shows squashfs for loop0 under ltsp running as well as under virtualbox.

But I found another approach solving this problem:
```
# set the running var
POST_SERVICE_SET_RUN_VAR="echo 'LTSP_RUN=1' >> /etc/environment"
```
sets an environment variable for the systems to boot in the ltsp.conf, only if ltsp is startet. A check shows:

under LTSP:
```
[ "${LTSP_RUN}" == 1 ] && echo 'yes' || echo 'no'
yes
```
under virtualbox of course:
```
[ "${LTSP_RUN}" == 1 ] && echo 'yes' || echo 'no'
no
```
I hope, this helps  in case somebody else is running into a problem like this.

Thank You, for taking time to help.

regards
jori

**Comment by alkisg** [comment, marked as answer]
I thought you meant "booted from the virtualbox-flat.vmdk image" or the "x86_64.img" image.
To see if you're booted in LTSP-client mode, use:

`test -d /run/ltsp/client`
